src/Country/Subdivision/CountrySubdivision.php

```diff
@@ -0,0 +1,5053 @@
+    case Argentina_Salta = 'AR-A';
+    case Argentina_Buenos_Aires = 'AR-B';
+    case Argentina_Ciudad_Autonoma_de_Buenos_Aires = 'AR-C';
+    case Argentina_San_Luis = 'AR-D';
+    case Argentina_Entre_Rios = 'AR-E';
+    case Argentina_La_Rioja = 'AR-F';
+    case Argentina_Santiago_del_Estero = 'AR-G';
+    case Argentina_Chaco = 'AR-H';
+    case Argentina_San_Juan = 'AR-J';
+    case Argentina_Catamarca = 'AR-K';
+    case Argentina_La_Pampa = 'AR-L';
+    case Argentina_Mendoza = 'AR-M';
+    case Argentina_Misiones = 'AR-N';
+    case Argentina_Formosa = 'AR-P';
+    case Argentina_Neuquen = 'AR-Q';
+    case Argentina_Rio_Negro = 'AR-R';
+    case Argentina_Santa_Fe = 'AR-S';
+    case Argentina_Tucuman = 'AR-T';
+    case Argentina_Chubut = 'AR-U';
+    case Argentina_Tierra_del_Fuego = 'AR-V';
+    case Argentina_Corrientes = 'AR-W';
+    case Argentina_Cordoba = 'AR-X';
+    case Argentina_Jujuy = 'AR-Y';
+    case Argentina_Santa_Cruz = 'AR-Z';
+    case Austria_Burgenland = 'AT-1';
+    case Austria_Karnten = 'AT-2';
+    case Austria_Niederosterreich = 'AT-3';
+    case Austria_Oberosterreich = 'AT-4';
+    case Austria_Salzburg = 'AT-5';
+    case Austria_Steiermark = 'AT-6';
+    case Austria_Tirol = 'AT-7';
+    case Austria_Vorarlberg = 'AT-8';
+    case Austria_Wien = 'AT-9';
+    case Australia_Australian_Capital_Territory = 'AU-ACT';
+    case Australia_New_South_Wales = 'AU-NSW';
+    case Australia_Northern_Territory = 'AU-NT';
+    case Australia_Queensland = 'AU-QLD';
+    case Australia_South_Australia = 'AU-SA';
+    case Australia_Tasmania = 'AU-TAS';
+    case Australia_Victoria = 'AU-VIC';
+    case Australia_Western_Australia = 'AU-WA';
+    case Azerbaijan_Abseron = 'AZ-ABS';
+    case Azerbaijan_Agstafa = 'AZ-AGA';
+    case Azerbaijan_Agcabdi = 'AZ-AGC';
+    case Azerbaijan_Agdam = 'AZ-AGM';
+    case Azerbaijan_Agdas = 'AZ-AGS';
+    case Azerbaijan_Agsu = 'AZ-AGU';
+    case Azerbaijan_Astara = 'AZ-AST';
+    case Azerbaijan_Baki = 'AZ-BA';
+    case Azerbaijan_Babk = 'AZ-BAB';
+    case Azerbaijan_Balakn = 'AZ-BAL';
+    case Azerbaijan_Brd = 'AZ-BAR';
+    case Azerbaijan_Beylqan = 'AZ-BEY';
+    case Azerbaijan_Bilsuvar = 'AZ-BIL';
+    case Azerbaijan_Cbrayil = 'AZ-CAB';
+    case Azerbaijan_Clilabad = 'AZ-CAL';
+    case Azerbaijan_Culfa = 'AZ-CUL';
+    case Azerbaijan_Dasksn = 'AZ-DAS';
+    case Azerbaijan_Fuzuli = 'AZ-FUZ';
+    case Azerbaijan_Gnc = 'AZ-GA';
+    case Azerbaijan_Gdby = 'AZ-GAD';
+    case Azerbaijan_Goranboy = 'AZ-GOR';
+    case Azerbaijan_Goycay = 'AZ-GOY';
+    case Azerbaijan_Goygol = 'AZ-GYG';
+    case Azerbaijan_Haciqabul = 'AZ-HAC';
+    case Azerbaijan_Imisli = 'AZ-IMI';
+    case Azerbaijan_Ismayilli = 'AZ-ISM';
+    case Azerbaijan_Klbcr = 'AZ-KAL';
+    case Azerbaijan_Kngrli = 'AZ-KAN';
+    case Azerbaijan_Kurdmir = 'AZ-KUR';
+    case Azerbaijan_Lnkran = 'AZ-LA';
+    case Azerbaijan_Lacin = 'AZ-LAC';
+    case Azerbaijan_Lnkran = 'AZ-LAN';
```

Comment:
@PrinsFrank What transliteration produces two `Azerbaijan_Lnkran`-s?

Reply:
![image](https://github.com/PrinsFrank/standards/assets/952007/d4a5f208-00c9-426e-b9cd-e6ab1295ea37)

Reply:
Don't we have a mechanism to add a suffix to the second one?

Reply:
We do, but that only works where different source scripts result in the same transliteration. The source scripts match in this case. I've decided to prefix with subdivision type to make the keys unique instead.